core/coreobjects/src/CMakeLists.txt

```diff
@@ -268,6 +268,7 @@ rtgen_config(${PROJECT_NAME}
 
 target_link_libraries(${LIB_NAME} PUBLIC daq::coretypes
                                          tsl::ordered_map
+                                         daq::security
```

Comment:
Security should then be in coreobjects. At least the permission manager part. Coreobjects should not link any opendaq library.